docs/diagrams/src/deployment.puml

```diff
@@ -0,0 +1,28 @@
+@startuml
```

Comment:
I think these diagrams should all be pulled for now, and recreated once we've reached a stable point with the eventing pipeline.